src-self-hosted/Module.zig

```diff
@@ -3623,3 +3678,125 @@ pub const ErrorMsg = struct {
 fn srcHashEql(a: std.zig.SrcHash, b: std.zig.SrcHash) bool {
     return @bitCast(u128, a) == @bitCast(u128, b);
 }
+
+fn bigIntAdd(allocator: *Allocator, lhs: Value, rhs: Value) !Value {
```

Comment:
This isn't really adding big ints necessarily right? It's adding 2 Values together, which may or may not be stored as BigInts.